megamek/src/megamek/client/ui/swing/CustomMechDialog.java

```diff
@@ -1226,6 +1277,11 @@ public void actionPerformed(ActionEvent actionEvent) {
                     a.liftOff(altitude);
                 }
             }
+            
+            if (isShip) {
```

Comment:
What about velocity?  This code would mean that you can't set a starting velocity.  Again, why can't this just be part of the previous `isAero` code?

Reply:
Fixed.